libp2p/transports/wstransport.nim

```diff
@@ -132,6 +134,18 @@ method start*(
   await procCall Transport(self).start(addrs)
   trace "Starting WS transport"
 
+  # use autotls if possible (and if cert/key not specified)
+  if not self.secure and not self.autotls.isNil():
+    try:
+      self.tlsCertificate = await self.autotls.getCertWhenReady()
+      self.tlsPrivateKey = self.autotls.getTLSPrivkey()
+    except AutoTLSError as exc:
+      raise newException(LPError, exc.msg, exc)
+    except TLSStreamProtocolError as exc:
+      raise newException(LPError, exc.msg, exc)
+    except CancelledError as exc:
+      raise newException(LPError, exc.msg, exc)
```

Comment:
why is CancelledError warapped to LPError error?

Reply:
As per conversation in DMs, I did it